src/register/Register.jsx

```diff
@@ -136,71 +171,92 @@ export default function Register() {
-                                    minLength={8}
-                                    maxLength={64}
-                                    onChange={validatePassword}
-                                    required
-                                />
-                            </p>
-                           {!isValidPassword && (
-                                <span className="error">Must be between 8-64 characters</span>
-                             )}
-                        </form-group>
-                        <div><br></br></div>
-                        <form-group controlId="formPassword">
-                            <p className="input-control">
+                <form-group controlId="formPassword">
+                  <p className="input-control">
+                    <label id="password">Password :</label>
+                    <input
+                      aria-labelledby="password"
+                      className="feilds"
+                      type={isPasswordShown ? "text" : "password"}
+                      name="password"
+                      placeholder="atleast 8 characters"
+                      minLength={8}
+                      maxLength={64}
+                      onChange={validatePassword}
+                      required
+                    />
+                  </p>
+                  {!isValidPassword && (
+                    <span className="error">
+                      Must be between 8-64 characters
+                    </span>
+                  )}
+                </form-group>
 
-                                <label id="confirmPassword">Confirm Password :</label>
-                                <input aria-labelledby="confirmPassword"
-                                    className="field"
-                                    type={isPasswordShown? "text" : "password"}
-                                    name="confirmPassword"
-                                    placeholder="Confirm Password"
-                                    minLength={8}
-                                    maxLength={64}
-                                    onChange={validateConfirmPassword}
-                                    required
-                                />
-                            </p>
-                            {!isValidConfirmPassword && (
-                                <span className="error">Must be same as password</span>
-                            )}
-                        </form-group>
+                <form-group controlId="formPassword">
+                  <p className="input-control">
+                    <label id="confirmPassword">Confirm Password :</label>
+                    <input
+                      aria-labelledby="confirmPassword"
+                      className="feilds"
+                      type={isPasswordShown ? "text" : "password"}
+                      name="confirmPassword"
+                      placeholder="atleast 8 characters"
```

Comment:
although the design shows this as placehloder, IMO it makes more sense if we show `re-type password` here instead since this field is not to insert a totally new password.